CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# v0.42.1 (_2019-10-21_)
+
+[Full Changelog](https://github.com/mozilla/application-services/compare/v0.42.0...v0.42.1)
+
+## Android
+
+### What's new
+
+- The Fennec bookmarks import method (`importBookmarksFromFennec`) will now return a list of pinned bookmarks.
```

Comment:
We should get back in the habit of linking to PRs in these...

Reply:
Oh right my bad!